### PR TITLE
INHEART-116 & INHEART-117: Add login screen elements to library & Create sign in page

### PR DESCRIPTION
**Pull request by pawelroguski**

### CODE REVIEW

File: apps/mobile/src/app/components/SignIn/SignInTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import { PixelRatio, StyleSheet } from 'react-native';
+
+import { Box, Image } from '@inheartive/ui/atoms';
+import { LoginFormControl } from '@inheartive/ui/molecules';
+import { View } from 'native-base';
+import logo from '../../assets/logo.jpg';
+
+/* eslint-disable-next-line */
+export interface LoginTemplateProps {}
```
Comment (zgr92) [inline apps/mobile/src/app/components/SignIn/SignInTemplate.tsx:11]: Can "disable" be removed now? We could remove empty props too as we discussed recently.

**Review by zgr92 — APPROVED** [review]
Just a detail fyi - in logo image file the heart is not the same color as the text, because I couldn't find the option and it was tmp fix ;)

File: apps/mobile/src/app/components/SignIn/SignInTemplate.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { PixelRatio, StyleSheet } from 'react-native';
+
+import { Box, Image } from '@inheartive/ui/atoms';
+import { LoginFormControl } from '@inheartive/ui/molecules';
+import { View } from 'native-base';
+import logo from '../../assets/logo.jpg';
```
Comment (krysia1) [inline apps/mobile/src/app/components/SignIn/SignInTemplate.tsx:8]: Didn't we want to use .png? 🤔

**Review by krysia1 — APPROVED** [review]

**Comment by krysia1** [comment]
I don't know why but when I run it on iOS the keyboard doesn't show up and I cannot unfocus input 😕 

https://user-images.githubusercontent.com/44640941/195641009-0fe90c52-db39-4301-ad5e-30d16867dcab.mov

**Review by KonradKarlinski — APPROVED** [review]

File: apps/mobile/src/app/components/SignIn/SignInTemplate.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+/* eslint-disable-next-line */
+export interface LoginTemplateProps {}
+
+export function SignInTemplate(props: LoginTemplateProps) {
+  console.log(logo);
```
Comment (konradhanus) [inline apps/mobile/src/app/components/SignIn/SignInTemplate.tsx:14]: - [] - remove console log

File: libs/ui/atoms/src/lib/Icons/Icons.tsx [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import { WarningOutlineIcon } from 'native-base';
```
Comment (konradhanus) [inline libs/ui/atoms/src/lib/Icons/Icons.tsx:1]: - [ ] - add unit test

File: .gitignore [hunk 5]
```diff
@@ -101,4 +101,4 @@ buck-out/
 **/vendor/bundle/
```
Comment (konradhanus) [inline .gitignore:101]: - [ ] - test

**Pull request merged**